src/protobuf-net/Meta/RuntimeTypeModel.cs

```diff
@@ -1609,6 +1643,34 @@ private void WriteAssemblyAttributes(CompilerOptions options, string assemblyNam
                     }
                 }
             }
+
+            WriteAssemblyInfoAttributes(options, asm);
+        }
+
+        private void WriteAssemblyInfoAttributes(CompilerOptions options, AssemblyBuilder asm)
+        {
+            WriteAssemblyInfoAttribute<AssemblyCompanyAttribute>(options, asm, options.AssemblyCompanyName);
+            WriteAssemblyInfoAttribute<AssemblyCopyrightAttribute>(options, asm, options.AssemblyCopyright);
+            WriteAssemblyInfoAttribute<AssemblyDescriptionAttribute>(options, asm, options.AssemblyDescription);
+            WriteAssemblyInfoAttribute<AssemblyProductAttribute>(options, asm, options.AssemblyProductName);
+            WriteAssemblyInfoAttribute<AssemblyTitleAttribute>(options, asm, options.AssemblyTitle);
+            WriteAssemblyInfoAttribute<AssemblyTrademarkAttribute>(options, asm, options.AssemblyTrademark);

```

Comment:
`AssemblyFileVersionAttribute` feels like it should be present; also, does the `AssemblyVersion` from the `AssemblyName` make it into `AssemblyVersionAttribute`?

Reply:
It does make it, so no need to duplicate efforts IMO.

Reply:
k, that would be a great thing to show in the tests - but `AssemblyFileVersionAttribute` is separate (a very common trick is to pin the `AssemblyVersion` to avoid binding redirect problems and use `AssemblyFileVersion` for the full sub-version info)

Reply:
Ah, you're right, only the AssemblyVersion gets added.

Reply:
Perhaps we should also have the AssemblyInformationalVersion attribute, for the project version.